streampipes-extensions/streampipes-connect-adapters-iiot/src/main/resources/org.apache.streampipes.connect.iiot.adapters.plc4x.modbus/documentation.md

```diff
@@ -26,8 +26,34 @@
 
 ## Description
 
-Shows the live position of the International Space Station (ISS), updated every two seconds.
-
+The Modbus adapter allows to connect to a PLC using the Modbus specification.
 
 ***
 
+## Configuration
+
+The following configuration options are available when creating the adapter:
+
+### PLC Address
+
+The IP address of the Modbus device without any prefix, which will be added automatically when creating the adapter.
+
+### PLC Port
+
+The PLC port refers to the port of the PLC, such as 502.
+
+### Node ID
+
+The Node ID refers to the ID of the specific device.
+
+### Nodes
+
+The `Nodes` section requires configuration options for the individual nodes.
+Nodes can be either imported from a comma-separated CSV file, or can be directly assigned in the configuration menu.
```

Comment:
Maybe we can here also provide a small sample CSV, that a user can copy to get started.

Reply:
Added a CSV example.